.github/workflows/sycl-check-ready-to-merge-prs.yml

```diff
@@ -0,0 +1,50 @@
+# This workflow checks for ready-to-merge PRs - if a PR is open, not a draft,
+# passed all checks, and has been approved, it will ping @intel/llvm-gatekeepers
+# if this group has not already been mentioned or if the last mention was more
+# than $days days ago.
+
+name: Check ready-to-merge PRs
+
+on:
+  schedule:
+    - cron: '0 * * * *' # every hour
+  workflow_dispatch:
+
+jobs:
+  notify-ready-prs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          # Number of days before repeating the gatekeepers ping
+          days=3
+          days_in_seconds=$((days*24*60*60))
+
+          # Function to ping gatekeepers and print debug info
```

Comment:
Looks like the function doesn't print the debug info. Should it be removed from the comment?